testproject/Assets/Scripts/CommandLineHandler.cs

```diff
@@ -3,8 +3,9 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 using Unity.Netcode;
+#if UNITY_UNET_PRESENT
 using Unity.Netcode.Transports.UNET;
-
+#endif
```

Comment:
remove this using directive completely?

Reply:
Since we have a bunch of manual tests that still use UNet, I decided to only include it and the associated code if UNet is present.  I need to go back through and update this to provide UnityTransport support too.